UnityPy/classes/AnimationClip.py

```diff
@@ -80,12 +80,12 @@ def UnpackFloats(
         indexPos: int = bitPos // 8
         bitPos %= 8
 
-        scale: float = 1.0 / self.m_Range
+        scale: float = 1.0 / self.m_Range if self.m_Range != 0 else 0.0001
```

Comment:
C#'s division by 0 results in infinity.
This isn't correctly reflected in your version.
```scale = (1.0 / self.m_Range) if self.m_Range else float("inf")``` would be better.

Reply:
Infinity is a better choice. I'll fix it